### PR TITLE
Deprecate Video Report components on frontend

### LINKED ISSUE #50: Deprecate "report" feature
Reports are redundant with the main mechanisms of Tournesol: We expect that bad quality videos if recommended by Tournesol will quickly receive negative comparison and will stop being recommended. Furthermore it is very difficult to properly handle reports from users, when most content slightly poilitcally involved will receive reports. We make efforts to have a robust and secure algorithm for aggregating comparisons. The reporting feature does not reach the same standard.

### PR DESCRIPTION
**Pull request by lfaucon**
Closes #50

### CODE REVIEW

**Comment by sergeivolodin** [comment]
Maybe we should convert all existing reports to comments?

File: frontend/src/constants.js [hunk 1]
```diff
@@ -41,44 +41,16 @@ export const featureColors = JSON.parse(`{
...
-    "misl1": "The content is very misleading",
-    "misl2": "The misinformation of the content is misleading and dangerous",
-    "hate": "The content contains hate speech",
-    "violence": "The content promotes violence",
-    "bully": "The content is cyber-bullying",
-    "threat": "The content contains threats",
-    "violent_act": "The content calls for violent acts",
-    "death_threat": "The content contains death threats"
-}`);
-
```
Comment (sergeivolodin) [inline frontend/src/constants.js:58]: The constants.js file is auto-generated from constants.py, so the change needs to be done there

**Comment by lfaucon** [comment]
Yes. I'll update the issue, mentionning that the feature has only been deprecated on the frontend, and that the second step of this issue (backend deprecation is less pressing and can be done later

File: frontend/src/constants.js (hunk 1 above)
Comment (lfaucon) [reply frontend/src/constants.js]: I removed my changes to this file

**Comment by lfaucon** [comment]
I'll deactivate my automatic code formatting for my next PRs. I reallize it is annoying for avoiding merge conflicts and reviewing code! 

This PR should be ready to be merged now

**Pull request merged**